src/kits/communityWebServer/StaticFileWeblet.sedona

```diff
@@ -0,0 +1,259 @@
+//
+// Copyright (c) 2018
+// Licensed under the Academic Free License version 3.0
+//
+// History:
+//   30 Jun 18  Vincent Wang  Static File Weblet
+//
+
+** 
+** StaticFileWeblet provides the ability to serve static files(html, css, js,
+** images etc) through http, so that user can use sedona as a minimal web
+** server and host a static web app just in sedona, without any outside
+** dependency. 
+** to save space on sedona, gzipped file is supported and I recommend to use it
+** since that will be more efficient(less bytes to be loaded and transferred)
+** 
+class StaticFileWeblet extends Weblet
+{
+////////////////////////////////////////////////////////////////
+// Weblet Registration
+////////////////////////////////////////////////////////////////
+
+  static void _sInit()
+  {
+    instance.register()
+  }
+  static internal inline StaticFileWeblet instance
+
+////////////////////////////////////////////////////////////////
+// Weblet
+////////////////////////////////////////////////////////////////
+
+  ** 
+  ** all static files to be served must be put under 'statics' folder
+  ** and the url will start with 'statics', too 
+  **
+  override Str prefix() { return "statics" }
```

Comment:
Does it make sense to make this prefix configurable? In other words, promote this as a property.

Reply:
good point, I will add a property slot for it